README.md

```diff
@@ -21,6 +20,22 @@ def deps do
 end
 ```
 
+Update deps
+
+```sh
+mix deps.get
+```
+
+Add the credentials to your config file (ex: `config.exs`)
+
+```elixir
+config :pluggy_elixir,
+  client_id: "your-app-client-id",
+  client_secret: "your-app-client-secret",
+```
+
+See more about configurations in `PluggyElixir.Config` docs.
```

Comment:
Can we reference the module file here?

Reply:
The `ex_docs` creates this ref when doc is built.